index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Dammy's Portfolio</title>
+ <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+ <!-- Header Section -->
+
+ <header>
+  <div id="page_logo">
+   <span>Dammy Shittu</span>
+  </div>
```

Comment:
- [ ] Please try to add logo in `<a>` tag. It will help the user to come back to the home page.